### ISSUE TITLE
Docusaurus 2

### ISSUE THREAD

**Issue by nainardev** [reduxjs/react-redux#1561]
Redux uses Docusaurus 2 but this uses its v1

**Comment by timdorr** [comment]
We know. We'll get around to it. There is a larger project in #1001.

**Comment by markerikson** [comment]
There's a couple different issues involved here:

- The Redux and RTK conversions were done by volunteers from the Docusaurus community
- The React-Redux site is the only one of the three that uses versioning, and I'm not sure of the status of that in DSv2

I'm not happy with how versioning works in DSv1, but I'm also not sure how it behaves in DSv2.  The differentiation between the main `/docs` folder and the versioned folders keeps confusing both contributors and us.

/cc @yangshun, @lex111 , @wgao19

**Comment by yangshun** [comment]
Versioning in v2 works similar enough. It's usable but maybe let's wait a while more :)

>  The differentiation between the main /docs folder and the versioned folders keeps confusing both contributors and us.

We hear you. We faced that problem ourselves. Recently we displayed the versions for the latest vs stable within the navigation bar (see https://v2.docusaurus.io/) and it seems to have helped reduce confusion. People know that there's a master version and would go to it when they edit something in `docs`.
